source/benchmarks/CMakeMacrosAddBenchmark.cmake

```diff
@@ -76,6 +76,9 @@ function (add_benchmark_for_api BASE_TARGET_NAME APPEND_API_TO_TARGET_NAME REGIS
         target_compile_definitions(${TARGET_NAME} PUBLIC USE_PIDFD)
     endif()
 
+
+	target_include_directories(${TARGET_NAME} PUBLIC ${SOURCE_ROOT})
```

Comment:
Could you explain why this is needed?  I seem to be able to build fine without it.